### PR TITLE
[helm] add support of loadBalancerIP for hub and router services

### PR DESCRIPTION
**Pull request by DANic-git**
### Description

Templates under  `charts/selenium-grid/templates/hub-service.yaml` and `charts/selenium-grid/templates/router-service.yaml` are updated to support loadBalancerIP value
Values under `charts/selenium-grid/values.yaml` is updated to support loadBalancerIP value
Readme under `charts/selenium-grid/README.md` is updated to describe new values

### Motivation and Context

This PR adds support to the `loadBalancerIP` parameter when the service type is [LoadBalancer](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer). This allows using of a specific IP for some cloud providers and on-premises

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by diemol** [comment]
@den-is any chance you could help us to review this PR?

**Review by diemol — CHANGES_REQUESTED** [review]
Could you please update your fork and also update the values in `Chart.yaml`?

**Comment by DANic-git** [comment]
> Could you please update your fork and also update the values in `Chart.yaml`?

Sorry for long answer
Updated from upstream

**Comment by DANic-git** [comment]
> Could you please update your fork and also update the values in `Chart.yaml`?

Values in `Chart.yaml` ?
Now is
```
apiVersion: v2
name: selenium-grid
description: A Helm chart for creating a Selenium Grid Server in Kubernetes
type: application
version: 0.6.2
appVersion: 4.3.0-20220628
icon: https://github.com/SeleniumHQ/docker-selenium/raw/trunk/logo.png
```
Do you mean up `version`?

**Comment by diemol** [comment]
Yes the version and also the CHANGELOG please, so we can do a release after merging.

**Comment by DANic-git** [comment]
> Yes the version and also the CHANGELOG please, so we can do a release after merging.

Done

File: charts/selenium-grid/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this helm chart will be documented in this file.
 
+## :heavy_check_mark: 0.7.1
```
Comment (diemol) [inline charts/selenium-grid/CHANGELOG.md]:
```suggestion
## :heavy_check_mark: 0.8.0
```

File: charts/selenium-grid/Chart.yaml [hunk 2]
```diff
@@ -2,6 +2,6 @@ apiVersion: v2
 name: selenium-grid
 description: A Helm chart for creating a Selenium Grid Server in Kubernetes
 type: application
-version: 0.7.0
+version: 0.7.1
```
Comment (diemol) [inline charts/selenium-grid/Chart.yaml]:
```suggestion
version: 0.8.0
```

**Review by diemol — APPROVED** [review]
Thank you, @germetist!

Minor changes, but I can do them myself.

**Pull request merged**